validation-test/stdlib/Lazy.swift.gyb

```diff
@@ -1279,5 +1279,172 @@ do {
   }
 }
 
-runAllTests()
+//===--- LazyPrefixWhile --------------------------------------------------===//
+
+let prefixDropWhileTests: [(data: [Int], value: Int, pivot: Int)] = [
+  ([],                       0,     0),
+  ([0],                      0,     0),
+  ([0],                     99,     1),
+  ([0, 10],                  0,     0),
+  ([0, 10],                 10,     1),
+  ([0, 10],                 99,     2),
+  ([0, 10, 20, 30, 40],      0,     0),
+  ([0, 10, 20, 30, 40],     10,     1),
+  ([0, 10, 20, 30, 40],     20,     2),
+  ([0, 10, 20, 30, 40],     30,     3),
+  ([0, 10, 20, 30, 40],     40,     4),
+  ([0, 10, 20, 30, 40],     99,     5) ]
+
+tests.test("LazyPrefixWhileIterator") {
+  let base = (1...10).lazy.prefix(while: { $0 < 4 })
+
+  var iter1 = base.makeIterator()
+  expectEqual(1, iter1.next())
+  expectEqual(2, iter1.next())
+  var iter2 = iter1.base
+  expectEqual(3, iter2.next())
+  expectEqual(4, iter2.next())
+
+  iter1 = base.makeIterator()
+  expectEqual(1, iter1.next())
+  expectEqual(2, iter1.next())
+  expectEqual(3, iter1.next())
+  expectEqual(nil, iter1.next())
+  iter2 = iter1.base
+  expectEqual(5, iter2.next())
+}
```

Comment:
@natecook1000 @gribozavr Did we get a resolution on this? I think I agree with Nate that the `base` property doesn't make sense in this context because we have no way to stuff the element back into the sequence.